### DISCUSSION TITLE
The 3D structure of the molecule is not displayed.

### DISCUSSION THREAD

**Discussion by okikosuke** [rdkit/rdkit#6605] [Q&A]
I want to display a 3D structure as shown in the following page, but it doesn't work. 3D structure is not displayed.
https://qiita.com/ottantachinque/items/80a22d2428231bcdfc74

```python
from rdkit import rdBase, Chem
from rdkit.Chem import AllChem, Draw
from rdkit.Chem.Draw import IPythonConsole
import sys, py3Dmol

smiles = "CN[C@H](C)[C@H](O)c1ccccc1"
mol = Chem.MolFromSmiles(smiles)
IPythonConsole.drawMol3D(mol)
```

I could get no image.

![image](https://github.com/rdkit/rdkit/assets/74843794/5421eacc-4e60-47d1-a4c0-83c8c25c339e)

However, I want 3d structure like this.

![image](https://github.com/rdkit/rdkit/assets/74843794/30b86be9-25f1-416e-aea0-d96baaa4a487)

- RDKit version: 2022.03.4
- OS: windows 10
- Python version (if relevant): 3.7.13
- py3dmol: 1.8.1
- Are you using conda? Yes

**Comment by ptosco** [comment]
All these basic RDKit usage topics are covered in by the [Getting Started with the RDKit in Python](https://www.rdkit.org/docs/GettingStartedInPython.html#getting-started-with-the-rdkit-in-python) document. You only need to search for "add 3D coordinates" in that web page.
